week-03/main.js

```diff
@@ -43,10 +60,31 @@ var list = [1, 2, 3, 4, 5];
 
 var list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+list.filter(function(num, i){
+  return i % 2 ===1;
```

Comment:
The filter function passes `num` and `i`. The order from the docs state that the first parameter is the element and the second is the index of that element.
Therefore your code is checking if the index, not the element is odd.